src/pages/Token/Index.tsx

```diff
@@ -0,0 +1,68 @@
+import {useParams} from "react-router-dom";
+import {gql, useQuery} from "@apollo/client";
+import {Stack, Grid} from "@mui/material";
+import React from "react";
+import HeaderSearch from "../layout/Search/Index";
+import TokenTitle from "./Title";
+import TokenTabs from "./Tabs";
+
+const TOKEN_QUERY = gql`
+  {
+    token_datas(where: {name_hash: {}, name: {_eq: "Aptos Zero: 7"}}) {
+      name
```

Comment:
Does the UI require all of these fields? It is preferable to load only what is needed.